kroxylicious-schema-validation/src/main/java/io/kroxylicious/proxy/filter/schema/validation/bytebuf/JsonFieldUniquenessWatcher.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.filter.schema.validation.bytebuf;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+
+class JsonFieldUniquenessWatcher implements JsonObjectWatcher {
```

Comment:
What's the difference between what you are doing here and using (https://fasterxml.github.io/jackson-core/javadoc/2.9/com/fasterxml/jackson/core/JsonParser.Feature.html#STRICT_DUPLICATE_DETECTION)

Reply:
d'oh, I missed that.